app/models/kit.rb

```diff
@@ -26,7 +26,7 @@ class Kit < ApplicationRecord
   scope :by_partner_key, ->(key) { joins(:items).where(items: { partner_key: key }) }
   scope :by_name, ->(name) { where("name ILIKE ?", "%#{name}%") }
 
-  validates :organization, :name, presence: true
+  validates :name, presence: true
```

Comment:
It `belong_to` an org, validation feels unneseccary.

Reply:
`belongs_to` used to imply optional - that switched to mandatory by default, so yep, a validation is redundant.